# Análisis de Usabilidad SUS - SGROAS
**Bloque C.3** | Fuente: `docs/mediciones/sus/sus-raw.csv` (n=10, P01..P10) | Método: Brooke 1996, IC95 con t de Student, escala adjetiva Bangor et al. 2009
Al abrir este cuaderno se ve el último run que produjo los números del documento (D.1). Rutas relativas a la raíz del repo. El CSV se lee como `latin-1` porque mezcla `Sí`/`Si` en consentimiento.

**Nota 2026-09-17:** este cuaderno mostraba salidas cacheadas de una versión anterior con n=15 (P01-P15). P11-P15 se retiraron del estudio SUS oficial por falta de respaldo documental verificable de sus respuestas (ver `dataset/sus/PARTICIPANTES-NO-INCLUIDOS.md`); las salidas de abajo se recalcularon sobre n=10 (P01-P10).

In [1]:
import csv, pathlib
raw = pathlib.Path("docs/mediciones/sus/sus-raw.csv").read_text(encoding="latin-1").splitlines()
rows = list(csv.DictReader(raw))
print(f"Participantes: {len(rows)}")
print("Codigos: " + ", ".join(r["codigo"] for r in rows))
scores = [float(r["sus_score"]) for r in rows]
print(f"Media verificada sus_score: {sum(scores) / len(scores):.1f}")


Participantes: 10
Codigos: P01, P02, P03, P04, P05, P06, P07, P08, P09, P10
Media verificada sus_score: 63.0


In [2]:
import json, pathlib
est = json.loads(pathlib.Path("docs/mediciones/sus/estadisticas-sus.json").read_text(encoding="utf-8"))
print(f"n={est['n']} media={est['media']:.1f} DT={est['desviacion_tipica']:.3f} SE={est['error_estandar']:.4f}")
print(f"IC95=[{est['ic95_inf']:.2f}; {est['ic95_sup']:.2f}] adjetiva={est['adjetiva']}")

n=10 media=63.0 DT=13.884 SE=4.3910
IC95=[53.07; 72.93] adjetiva=Bueno


## Figuras (se renderizan al abrir, sin re-ejecutar)
![SUS por participante](../docs/mediciones/sus/figuras/fig-sus-por-participante.png)
![Respuestas por ítem](../docs/mediciones/sus/figuras/fig-sus-item-respuestas.png)
Generadas por `scripts/sus-analysis.py`. Detalle por participante en `P01.json`..`P10.json` y `ANALISIS-SUS.md`.
## Lectura
Media 63.0 (n=10, IC95 [53.07; 72.93]), por debajo del umbral de 70: usabilidad *Bueno* en la escala adjetiva pero zona *marginal*, consistente con cap1/cap2/cap8/cap9/cap12 del informe.